fitsio/header.py

```diff
@@ -407,7 +407,7 @@ def _record2card(self, record):
         elif name == 'CONTINUE':
             card = 'CONTINUE   %s' % value
```

Comment:
The header object won't contain any CONTINUE, because the long string is kept and the continue discarded.

One might consider it a bug that there is a check for CONTINUE at all, since that code is unreachable.

This printout does't break up the strings because I didn't want to reimplement all the code that makes CONTINUE work in cfitsio.

The printing of a header was not meant to be an exact representation of the header as it would look on disk; there may be other issues beyond what this PR fixes (and this CONTINUE thing)

If we want CONTINUE to be used exactly the same as cfitsio would do that would take more work, which I consider out of scope for this PR, but I'm happy to consider it in another issue.